src/content/app/genome-browser/state/track-panel/trackPanelSelectors.ts

```diff
@@ -58,3 +64,24 @@ export const isTrackCollapsed = (state: RootState, trackId: string) => {
   const trackPanel = getActiveTrackPanel(state);
   return trackPanel.collapsedTrackIds.includes(trackId);
 };
+
+export const getTrackPanelState = (): TrackPanelState => {
+  const genomeId = browserStorageService.getActiveGenomeId();
+  return genomeId ? { [genomeId]: getTrackPanelStateForGenome(genomeId) } : {};
+};
+
+export const getTrackPanelStateForGenome = (
+  genomeId: string
+): TrackPanelStateForGenome => {
+  return genomeId
+    ? {
+        ...defaultTrackPanelStateForGenome,
+        ...getPersistentTrackPanelStateForGenome(genomeId)
+      }
+    : defaultTrackPanelStateForGenome;
+};
+
+export const getPersistentTrackPanelStateForGenome = (
+  genomeId: string
+): Partial<TrackPanelStateForGenome> =>
+  browserStorageService.getTrackPanels()[genomeId] || {};
```

Comment:
I am sorry, I mislead you in my previous comment. I only looked at the names of these functions, and didn't catch that they all are looking something up in the browser storage rather than in the redux state. They are not selectors at all 😞 Could you please move them back the the slice file? My apologies.

Reply:
No problem. I've put them back.